src/test/java/org/opensearch/security/auditlog/AbstractAuditlogiUnitTest.java

```diff
@@ -38,7 +38,7 @@ protected String getResourceFolder() {
         return "auditlog";
     }
 
-    protected final void setup(Settings settings) throws Exception {
```

Comment:
Just realized that there is a typo in the name of this class. It should be renamed to `AbstractAuditlogUnitTest`